libqtile/core/manager.py

```diff
@@ -50,12 +51,75 @@
 from .. import hook
 from .. import utils
 from .. import window
-from libqtile.backend.x11 import xcbq
-from libqtile import command_interface
-from libqtile.command_client import InteractiveCommandClient
-from libqtile.command_interface import QtileCommandInterface, IPCCommandServer
-from libqtile.command_object import CommandObject, CommandError, CommandException
-from libqtile.lazy import lazy
+from ..backend.x11 import xcbq
+from .. import command_interface
+from ..command_client import InteractiveCommandClient
+from ..command_interface import QtileCommandInterface, IPCCommandServer
+from ..command_object import CommandObject, CommandError, CommandException
+from ..lazy import lazy
+from ..confreader import Config, ConfigError
+from ..xkeysyms import keysyms
+from ..backend.x11.xcore import XCore
```

Comment:
Do we really want to favor relative imports over absolute imports? IIRC, PEP8 recommends absolute imports, and I personally prefer them also.

Reply:
I personally encountered import issues (module not initialized ot similar) with absolute imports, that's why I changed them to relative ones. Maybe it was just misconfiguration on my side. If absolute imports are preferred, lets use them! Either way, we should be consistent.

Reply:
I'm using relative imports in all my projects, I thought I had read somewhere they were best-practice... either I misread, or the best-practices have changed :smile: Searched again, and indeed there are more pros than cons for absolute imports. Will fix all my projects haha

Reply:
Let's keep this PR atomic and stick to how it was before, we'll go through all imports at once later. I'm opening an issue for this.

Reply:
Changed back to absolute imports